subsys/bluetooth/controller/Kconfig

```diff
@@ -1004,14 +1004,6 @@ config BT_CTLR_CONN_ISO_PDU_LEN_MAX
 	help
 	  Maximum Connected Isochronous Channel PDU Length.
 
-config BT_CTLR_CONN_ISO_SDU_LEN_MAX
-	int "Maximum Connected Isochronous Channel SDU Length"
-	depends on BT_CTLR_CONN_ISO
-	range 1 4095
-	default 251
-	help
-	  Maximum Connected Isochronous Channel SDU Length.
```

Comment:
Do we need to go through the deprecation process for this?

Reply:
ISO is experimental upstream. And not sure if any downstream implementation use this.

Reply:
Ideally we would `select EXPERIMENTAL` for all experiment controller Kconfigs as well then